README.md

```diff
@@ -392,6 +392,25 @@ foo.yaml                     # -> Config-file ; Copy-on-apply
 foo.yaml.appcli.schema.json  # -> Schema-file ; Ignore-on-apply
 ```
 
+### Secrets management
+
+Sensitive values can be encrypted inside the `settings.yml` file and then decrypted during deployment.
+
+```bash
+# Automatically encrypt and set.
+./myapp configure set -e "path.to.field" "my-secret-value"
+
+# Manually encrypt and set.
+./myapp encrypt "my-secret-value" # Returns `enc:id=X:....`
+./myapp configure edit # Set the above value to the field.
+```
+
+Any secret values set will be automatically decrypted by APPCLI (using the `conf/key` key)
+to temporary files just before deployment.
```

Comment:
No this is not correct.
See my comment here https://brightsparklabs.atlassian.net/browse/SYS-120?focusedCommentId=59019

Rework this to detail the workflow properly then merge.